### PR TITLE
User story 9

### PR DESCRIPTION
**Pull request by Joshua-Tukman**
I cleaned up a couple small errors that I overlooked in the previous branches. Lets just merge this one and not user_stories 6-8

### CODE REVIEW

File: spec/features/merchant_user/navigation_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                                       city: "Los Angeles",
+                                       state: "California",
+                                       zip: "90210",
+                                       email: "employee_email@gmail.com",
+                                       password: "employee_password",
+                                       role: 1) 
+                                
+            allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(employee)  
+
+            visit '/admin/dashboard'
```
Comment (JPuchalla) [inline spec/features/merchant_user/navigation_spec.rb:18]: Solid tests, and great work on the implementation of admin access on carts.

File: spec/features/default_user/navigation_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'As a default user' do
+    describe 'site navigation' do
+        it 'I am not able to access any path that begins with /merchant, or /admin' do
```
Comment (melatran) [inline spec/features/default_user/navigation_spec.rb:5]: Tests are great. Thanks for testing out both scenarios

**Comment by melatran** [comment]
The tests looks great. They test for multiple scenarios depending on the user which is fantastic.

**Comment by Rostammahabadi** [comment]
approve!

**Pull request merged**